### DISCUSSION TITLE
Can I Run EP While Running Start Menu X Pro?

### DISCUSSION THREAD

**Discussion by paulp575** [valinet/ExplorerPatcher#3856] [Q&A]
I would like to restore the Windows File Explorer's ribbon on my Windows 11 Pro laptop.
I'm concerned it will conflict with Start Menu X Pro (version 7.77) which I'm running to get the Windows 7 style of the Start Menu.
Can I install EP and turn off all EP modifications to the Start Menu so there will be no conflicts?
Thanks.

**Comment by pyrates999** [comment]
You can install openshell to get the windows 7 style start menu and that doesn't conflict with EP: https://github.com/Open-Shell/Open-Shell-Menu

**Reply by paulp575** [reply]
Thanks, but I've already paid for Start Menu X Pro and it works for what I need.
I just need to know if EP will work while I'm running Start Menu X Pro.
Please don't suggest alternatives; just let me know if I can run both or not.
Thanks!

**Reply by pyrates999** [reply]
Not sure.  You can install EP and find out.
